tests/utils/whitelist.test.js

```diff
@@ -0,0 +1,74 @@
+const { checkWhiteList } = require('../../js/utils/isWhiteList');
```

Comment:
Please use the same naming convention as mentioned in the above set of comments.